### ISSUE TITLE
footer with version & date

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/limelight#143]
it's very useful to have a footer on every page of the workshop that shows what version it is & the date of the version.

**Comment by rohrlach** [comment]
Sure - Will have to think about the best way to do this such that someone doesn't have to remember to modify it each time

**Comment by helenvarley** [comment]
yes, it should be automated. i think we almost achieved that before but then it didn't always happen somehow ...
